vapi/simulator/simulator.go

```diff
@@ -306,12 +306,6 @@ func Status(w http.ResponseWriter, httpStatus int, val ...interface{}) {
 	}
 }
 
-// StatusOK responds with http.StatusOK and json encoded val if given.
-// For use with "/api" endpoints.
-func StatusOK(w http.ResponseWriter, val ...interface{}) {
-	Status(w, http.StatusOK, val...)
-}
-
 // OK responds with http.StatusOK and json encoded val if given.
 // For use with "/rest" endpoints where the response is a "value" wrapped structure.
 func OK(w http.ResponseWriter, val ...interface{}) {
```

Comment:
nit: perhaps in the future we can revisit this API as it‘s confusing to have StatusOK and OK functions. I understand from the comment that the difference is in the response wrapping.

Reply:
Agreed!